4221 improvements to dedicated docs (#4249)
platformsh/platformsh-docs: sites/platform/src/dedicated-environments/backups-restores.md

---
title: "Dedicated backup and restores"
weight: 1
sidebarTitle: "Dedicated backups"
layout: single
description:  "Backups are retained for different periods depending on various factors and whether you’re using a Dedicated Gen 2 or Dedicated Gen 3 Environment. These processes can be either manual or automated."
---

{{% description %}}

## Dedicated Generation 2 Backups

Platform.sh takes a byte-for-byte snapshot of Dedicated Gen 2 production Environments every 6 hours. Backups are retained for different durations depending on when they were taken. 

> |When taken      |Retention            | 
> |----------------|---------------------|
> | Days 1–3       | Every backup        |  
> | Days 4–6       | One backup per day  |   
> | Weeks 2–6      | One backup per week |
> | Weeks 8–12     | One bi-weekly backup|   
> | Weeks 12–22    | One backup per month|  

Reviewer: ```suggestion
| Backup retention period     | Retention            |  # of backups from time period available |
|----------------|---------------------|--------------------|
| 1–3 days ago          | Every backup taken in that period | 6 (1 day) to 18 (3 days) |  
| 4-6 days ago         | One backup per day in that period  |   1 (4 days) to 3 (6 days) |
| 2–6 weeks ago      | One backup per week | 1 (2 weeks) to 5 (6 weeks) |
| 8–12 weeks ago   | One bi-weekly backup | 1 (8 weeks) to 3 (12 weeks) |    
| 12–22 weeks ago    | One backup per month | 1 (12 weeks) to 3 (22 weeks)  |

Put another way, the number of backups available for restoration depending on the age of the Dedicate environment might look as follows:

|  Account age    | last 72 hours  | 4-6 days ago | 2-6 weeks ago | 8-12 weeks ago | 12-22 weeks ago | Total |
|----------------|---------------|--------------|----------------|-----------------|-------------------|-----|
| 1 week               |         18            |        3             |          0                |            0              |              0                |   21  |
| 12 weeks           |         18            |        3              |            5            |         3                   |               0               |  29  |
| 52+ weeks        |         18            |        3              |            5            |         3                  |           3                    |   32 |
```

---
title: "Dedicated backup and restores"
weight: 1
sidebarTitle: "Dedicated backups"
layout: single
description:  "Backups are retained for different periods depending on various factors and whether you’re using a Dedicated Gen 2 or Dedicated Gen 3 Environment. These processes can be either manual or automated."
---

{{% description %}}

## Dedicated Generation 2 Backups

> Platform.sh takes a byte-for-byte snapshot of Dedicated Gen 2 production Environments every 6 hours. Backups are retained for different durations depending on when they were taken. 

Reviewer: @gilzow you had reviewed that the "when they were taken" phrasing in the table wasn't the best. What do you think of this? https://4221-improvements-to-jluuz2y-ucq44jg6ofare.eu-5.platformsh.site/dedicated-environments/backups-restores.html#dedicated-generation-2-backups

| Backup retention period     | Retention            |  # of backups from time period available |
|----------------|---------------------|--------------------|
| 1–3 days ago          | Every backup taken in that period | 6 (1 day) to 18 (3 days) |  
| 4-6 days ago         | One backup per day in that period  |   1 (4 days) to 3 (6 days) |
| 2–6 weeks ago      | One backup per week | 1 (2 weeks) to 5 (6 weeks) |
| 8–12 weeks ago   | One bi-weekly backup | 1 (8 weeks) to 3 (12 weeks) |    
| 12–22 weeks ago    | One backup per month | 1 (12 weeks) to 3 (22 weeks)  |

> Put another way, the number of backups available for restoration depending on the age of the Dedicate environment might look as follows:

Reviewer: ```suggestion
Put another way, the number of backups available for restoration depending on the age of the Dedicated environment might look as follows:
```

> Put another way, the number of backups available for restoration depending on the age of the Dedicate environment might look as follows:

Reviewer: ```suggestion
Put another way, the number of backups available for restoration - in each of the backup age groupings described above - might look as follows for a few Dedicated environment ages:
```

> Put another way, the number of backups available for restoration depending on the age of the Dedicate environment might look as follows:

Reviewer: ```suggestion
Put another way, the number of backups available for restoration - in each of the backup age groupings described above - might look as follows for a few Dedicated environment ages:
```

> |  Account age    | last 72 hours  | 4-6 days ago | 2-6 weeks ago | 8-12 weeks ago | 12-22 weeks ago | Total |

Reviewer: ```suggestion
|  Environment age    | 72 hours old or younger  | 4-6 days old | 2-6 weeks old | 8-12 weeks old | 12-22 old  | Total |
```
